### ISSUE TITLE
Please add a link to documentation on the template engine at https://readwise.io/export/obsidian/preferences

### ISSUE THREAD

**Issue by thomasvs** [readwiseio/obsidian-readwise#15]
I am assuming it's jinja2 but I'm not 100% certain, and it would be helpful to add a link on the page.

**Comment by tadeoos** [comment]
When you click on the "Documentation" link next to the template section header you're transferred to https://help.readwise.io/article/126-how-can-i-customize-the-readwise-to-obsidian-export page where it's specified that it is indeed jinja2

**Comment by thomasvs** [comment]
Thanks for confirming it's jinja2.  Still, it would be much more helpful to include that note on the actual page where you're supposed to put the templates, to make that page self-documenting.

In addition, having a link to it would be very helpful.

I think I figured out that https://jinja2docs.readthedocs.io/en/stable/templates.html is the best starting point.

But if you google jinja2, none of the top links point there.  The first link points to jinja3, and the second link points to https://jinja2docs.readthedocs.io/en/stable/ which is a terrible starting point for template writers, as the intended audience is application developers and it starts by explaining how to install jinja2 and how to call the API from Python, which is not relevant for template writers.

The link I found is buried deep in that table of contents.  You'd do your users a favor by directly linking to the right template guide.
